config/routes.rb

```diff
@@ -16,6 +16,8 @@
     post 'i/signup/student', action: :student_signup, as: :newflow_signup_post
     get 'i/signup/educator', action: :educator_signup_form, as: :educator_signup
     post 'i/signup/educator', action: :educator_signup, as: :educator_signup_post
+    get 'i/signup/educator_complete', action: :educator_complete_form, as: :educator_complete
+    post 'i/signup/educator_complete', action: :educator_complete, as: :educator_complete_post
```

Comment:
This works, but it'd be better if they were nested under `/educator/` like so: `'i/signup/educator/complete'`